Assets/HoloToolkit/Common/Shaders/Standard.shader

```diff
@@ -261,6 +262,7 @@ Shader "MixedRealityToolkit/Standard"
 #endif
 
 #if defined(_HOVER_LIGHT)
+			fixed _HoverLightRadius;
```

Comment:
@quakeulf thank you for taking a look at this. I love the idea of making the gradient exponential. I am a bit curious about why you needed to add this since the radius can be controlled on the HoverLight component (it is packed into the 'w' component of the hover light data). Is this so the radius can be controlled on a per material basis?

Reply:
Well, I wanted more options on the shader itself, and I per-material is a bonus I didn't think of. That's about it I can think of for reasons right now.